pkg/validation/spec/gnostic_test.go

```diff
@@ -55,7 +55,7 @@ func gnosticCommonTest(t testing.TB, fuzzer *fuzz.Fuzzer) {
 	ok, err := actual.FromGnostic(gnosticSpec)
 	require.NoError(t, err)
 	require.True(t, ok)
-	if !reflect.DeepEqual(expected, actual) {
+	if !cmp.Equal(expected, actual, SwaggerDiffOptions...) {
```

Comment:
curious about this change... did this test fail. without this? same question below

Reply:
Yes, because the new reference has unexported fields, so you need to compare refs as a string.

Reply:
That's why I made OP run the tests in k/k, which seemed to have passed. I can't think of a place where we directly compare them?